tests/bluetooth/bsim/audio/src/tbs_test.c

```diff
@@ -37,13 +48,42 @@ static bool tbs_authorize_cb(struct bt_conn *conn)
 	return conn == default_conn;
 }
 
+static void tbs_terminate_call_cb(struct bt_conn *conn, uint8_t call_index,
+				  uint8_t reason)
+{
+	printk("Terminating call with id %u reason: %u", call_index, reason);
+	call_terminated = true;
+	call_placed = false;
+}
+
+static void tbs_accept_call_cb(struct bt_conn *conn, uint8_t call_index)
+{
+	printk("Accepting call with index %u\n", call_index);
+	call_accepted = true;
+}
+
+static void tbs_retrieve_call_cb(struct bt_conn *conn, uint8_t call_index)
+{
+	printk("Retrieve call with index %u\n", call_index);
+	call_retrieved = true;
+}
+
+static void tbs_join_calls_cb(struct bt_conn *conn,
+			      uint8_t call_index_count,
+			      const uint8_t *call_indexes)
+{
```

Comment:
should we add a flag here to signal that the join cb was called?

Reply:
Done